api/src/main/java/gov/cms/ab2d/api/controller/OktaProxyAPI.java

```diff
@@ -0,0 +1,38 @@
+package gov.cms.ab2d.api.controller;
+
+import gov.cms.ab2d.api.security.OktaTokenGenerator;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import static gov.cms.ab2d.common.util.Constants.OKTA_PROXY_ENDPOINT;
+
+@Slf4j
+@RestController
+@RequestMapping(produces = "application/json")
+public class OktaProxyAPI {
+
+    @Autowired
+    private OktaTokenGenerator oktaTokenGenerator;
+
+    // This API endpoint does not have authentication, an exception was made in SecurityConfig.java and JWTAuthenticationFilter.java
+    @CrossOrigin // test server for the sandbox runs from port 4000
+    @ResponseStatus(value = HttpStatus.OK)
+    @PostMapping(OKTA_PROXY_ENDPOINT)
+    public ResponseEntity<OktaProxyResponse> getOktaToken(@RequestParam String clientID, @RequestParam String clientSecret) throws URISyntaxException,
```

Comment:
These come within POST body, not query string, correct?

Reply:
Correct it's coming from the post body 
<img width="1680" alt="Screen Shot 2020-03-31 at 11 21 02 AM" src="https://user-images.githubusercontent.com/153646/78050563-06f55a80-7342-11ea-834d-ac2c1ba5ddd1.png">